pkg/sentry/platform/kvm/machine_arm64_unsafe.go

```diff
@@ -47,6 +48,16 @@ func (m *machine) initArchState() error {
 		uintptr(unsafe.Pointer(&vcpuInit))); errno != 0 {
 		panic(fmt.Sprintf("error setting KVM_ARM_PREFERRED_TARGET failed: %v", errno))
 	}
+
+	// initialize all vCPUs
+	m.initialvCPUs = make(map[int]*vCPU)
+	for int(m.nextID) < m.maxVCPUs-1 {
+		m.mu.Lock()
+		c := m.newVCPU()
```

Comment:
I think we need to add a comment that will explain why we create vcpu here and why it is different with x86.